components/badge/badge.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Badge|Sapota UI </title>
+    <link rel="stylesheet" href="../../docs/css/index.css" />
+    <link rel="stylesheet" href="../Avatar/avatar.css" />
+    <link rel="stylesheet" href="./badge.css" />
+    <style>
+        .avatar-container, .container-badge-icon{
+            padding: 1rem;
+            margin: auto;
+            max-width: 500px;
+            display: flex;
+            justify-content: center;
+        }
+        .badge-box i{font-size: 3.2rem;}
+
+    </style>
+</head>
+<body>
+    <div class="wrapper">
+        <div class="container-bdage-avatar">
```

Comment:
container-bdage-avatar ,  there is a typo i think

Reply:
thanks for pointing it out